### ISSUE TITLE
Unexpected lemmatization for "it" / "its"

### ISSUE THREAD

**Issue by davechallis** [explosion/spaCy#906]
Basic example:

    import spacy
    nlp = spacy.load('en')
    for token in nlp("IT it ITS its"):
        print(token, token.lemma_)

Output:

    IT it
    it -PRON-
    ITS it
    its -PRON-

Expected:

    IT it
    it it
    ITS it
    its it

## Info about spaCy

* **spaCy version:** 1.7.2
* **Platform:** Darwin-16.4.0-x86_64-i386-64bit
* **Python version:** 3.6.0
* **Installed models:** en

**Comment by honnibal** [comment]
This is the expected behaviour --- the 1.6 behaviour was a regression.

If you find the current behaviour particularly inconvenient, could you describe why? There seems to me no really satisfying solution for the lemma for pronouns, but I understand that many don't like the current policy.

**Comment by ddofer** [comment]
Well, it's not what some users might expect naively from lemmatization ("let's find the root meaning/word"). It can also have a fair amount of info loss, e.g. "they" vs "I". 
(I came acorss this thread while trying to understand where the "-PRON-" in my lemmatized text came from, I thought i'd messed it up).

**Comment by davechallis** [comment]
I wouldn't say that I found it inconvenient, it was more that the output was unexpected, so thought it was a bug rather than intended (i.e. I thought the lemmatizer would always return a word, not a part of speech).

I'm not a linguist, so don't know what the correct lemma is, this is the only really unexpected thing I encountered when moving from NLTK's lemmatization to Spacy's, so perhaps it's just a documentation issue.
